danieltichiyama/exercise-app
# Pwa setup 2

Does everything that PWA-setup does plus:

-  after npm run build and going to localhost:8080/nutrition you can kill your docker-compose (cmd/ctrl + c) and refresh the page, the dynamically loaded content will disappear but the requests to the server will be saved.

-  coming soon: pwa-setup 3, saving data locally to populate into the user's interface, even when offline... oh gawd.

pwa-setup info below:
PWA initial setup complete, minimal functionality.
To test:

docker-compose down -v && docker-compose up,
npm run build,
in an INCOGNITO tab, go to localhost:8080,
Page loads normally, all app links should take you to the registration page,
Login with calmoon808@gmail.com,
App should automatically take you to the page you were on (needs to be fixed)
open dev tools, Application, Service Worker, select 'Offline',
all pages should reload but no data will be populated (this is the minimal part...)
optional:

download and setup android studio, set up an AVD emulator,
open site in chrome
click on three dots and click Add website to home screen
should show basic fitworks logo and the app should be available on home screen
opening on home screen will open a splash screen followed by the app itself
app should not have an address bar or other chrome browser window functions visible.
